src/components/hacs-checkbox.ts

```diff
@@ -16,13 +16,20 @@ export class HacsCheckbox extends LitElement {
   @property({ attribute: false }) public label: string;
   @property({ attribute: false }) public id: string;
 
+  private get _checkboxClass(): ClassInfo {
+    return {
+      'checkbox': true,
+      'checked': this.checked,
+    };
+  }
+
   protected render(): TemplateResult | void {
     return html`
       <div class="checkbox-container">
-        <div class="checkbox" @click=${this._checkboxClicked}>
+        <div class=${classMap(this._checkboxClass)} @click=${this._checkboxClicked}>
```

Comment:
```suggestion
        <div ?checked=${this.checked} @click=${this._checkboxClicked}>
```

Reply:
Why, though? `checked` is not a standard div attribute. If it was `<input type="checkbox">` yeah, but IMO this is a worse solution.

We can do it with `data-checked`, but why not just pure native CSS?